src/server/scripts/Northrend/Ulduar/Ulduar/boss_hodir.cpp

```diff
@@ -228,6 +228,8 @@ class boss_hodir : public CreatureScript
 
             if (pInstance && pInstance->GetData(TYPE_HODIR) != DONE)
                 pInstance->SetData(TYPE_HODIR, NOT_STARTED);
+            if (GameObject* go = me->FindNearestGameObject(GO_HODIR_FRONTDOOR, 300.0f))
+                    go->SetGoState(GO_STATE_ACTIVE);
```

Comment:
add brackets to the `if` statement, like:
```C++
            if (GameObject* go = me->FindNearestGameObject(GO_HODIR_FRONTDOOR, 300.0f))
           {
                    go->SetGoState(GO_STATE_ACTIVE);
            }
```

Reply:
done